department-of-veterans-affairs/abd-vro
# Update QA BGS URL

## What was the problem?

Invalid URL for VA BGS environment in QA

Associated tickets or Slack threads:
- #2324 

## How does this fix it?[^1]

* Prevents log buffering in the application to avoid confusion
* Updates the URL for the QA environment for BGS

## How to test this PR
- I tested it through a deploy to the QA environment, and saw success.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
